wiki/Reporting_Bad_Behaviour/Handling_Cheating/en.md

```diff
@@ -1,74 +1,72 @@
-Why am I banned?
-----------------
+# Handling Cheating
 
--   10 silences issued in your userpage.
--   You gave your account to somebody else.
+
+## Why am I banned?
+Users are getting banned for several reasons not limited to:
```

Comment:
replace this whole sentence with "A user may get banned if they do any of the following" - "are getting banned" is not a very positive prospect